modules/widgets/wordpress-post-widget.php

```diff
@@ -64,6 +64,7 @@ public function get_site_info( $site ) {
 	 * Set up the widget display on the front end
 	 */
 	public function widget( $args, $instance ) {
+		/** This filter is documented in core/src/wp-includes/default-widgets.php */
```

Comment:
Noting that we can use this once shipped for folks who ask how to remove the URL/site title from the widget.